platformio.ini

```diff
@@ -115,7 +115,7 @@ lib_deps =
   https://github.com/boschsensortec/Bosch-BSEC2-Library#v1.5.2400
   boschsensortec/BME68x Sensor Library@^1.1.40407
   adafruit/Adafruit MCP9808 Library@^2.0.0
-  https://github.com/Tinyu-Zhao/INA3221@^0.0.1
+  https://github.com/KodinLanewave/INA3221#main
```

Comment:
Can we do a version tag instead? Pegging it to a branch makes for non-idempotent builds potentially.
We may actually want to fork this one into the meshtastic org at some point.

Reply:
@thebentern sure, I'm just way less familiar with tags and couldn't get my build environment to utilize them when I set that up.  I'll see what I can do though.